src/lib/provable/gadgets/elliptic-curve.ts

```diff
@@ -236,7 +236,8 @@ function verifyEcdsaGeneric(
     G?: { windowSize: number; multiples?: Point[] };
     P?: { windowSize: number; multiples?: Point[] };
     ia?: point;
-  } = { G: { windowSize: 4 }, P: { windowSize: 4 } }
+  } = { G: { windowSize: 4 }, P: { windowSize: 4 } },
+  checkSCanonical = false
```

Comment:
I went with this extra flag to stay backwards compatible.
This breaks `ecdsa.verify` yet again, we still need to figure out how we want to handle this